nengo/neurons.py

```diff
@@ -260,6 +311,14 @@ def gain_bias(self, max_rates, intercepts):
         bias = 1 - gain * intercepts
         return gain, bias
 
+    def max_rates_intercepts(self, gain, bias):
+        """Compute the inverse of gain_bias."""
+        intercepts = (1 - bias) / gain
+        max_rates = 1.0 / (self.tau_ref - self.tau_rc * np.log1p(
+            1.0 / (gain * (intercepts - 1) - 1)))
+        max_rates = np.nan_to_num(max_rates)
```

Comment:
`max_rates` will be non-finite if `-1 < gain * (intercepts - 1) - 1 < 0`.  That can happen if `intercepts >= 1` and `gain` is some smallish value, or if intercepts are in the normal (-1,1) range it only happens if `gain` is 0.  In either of those cases (`intercepts >= 1` or `gain==0`) the firing rate will be zero, so converting `nan` to 0 seemed reasonable.  

I probably wouldn't just return `nan` and let the caller deal with it, since it may not be obvious to them why they're getting `nans`.  We could detect those conditions and raise a warning/exception though, if people think that is preferable to 0.